android/src/main/java/com/swmansion/rnscreens/gamma/tabs/TabsHost.kt

```diff
@@ -482,6 +498,11 @@ class TabsHost(
         }
     }
 
+    override fun requestLayout() {
+        super.requestLayout()
+        refreshLayout()
+    }
```

Comment:
ay ay, we're really asking for troubles here 😅 I'll try to debug it in a few a bit, to see whether we can do better.